Source/cursor.cpp

```diff
@@ -264,8 +264,9 @@ void CheckCursMove()
 			sx += GetScreenWidth() / 4;
 		}
 	}
-	if (sy > GetMainPanel().position.y - 1 && MousePosition.x >= GetMainPanel().position.x && MousePosition.x < GetMainPanel().position.x + PANEL_WIDTH && track_isscrolling()) {
-		sy = GetMainPanel().position.y - 1;
+	auto &mainPanel = GetMainPanel();
+	if (sy > mainPanel.position.y - 1 && MousePosition.x >= mainPanel.position.x && MousePosition.x < GetMainPanel().position.x + mainPanel.size.width && track_isscrolling()) {

```

Comment:
```suggestion
	if (mainPanel.Contains(MousePosition) && track_isscrolling()) {
```

Reply:
I modified so many lines that I didn't thought about this one. This make it much more simple. 🙂